examples/webpack/tsconfig.json

```diff
@@ -0,0 +1,41 @@
+{
+  "compilerOptions": {
```

Comment:
this doesn't look like a recent tsconfig. Did you make sure to use the latest generator?

Reply:
yes, 5.0.8 is the latest version.

I updated manually.